sim_pipeline/gg_lens.py

```diff
@@ -81,15 +81,24 @@ def validity_test(self, min_image_separation=0, max_image_separation=10):
 
         kwargs_lens = kwargs_params['kwargs_lens']
         image_positions = lens_eq_solver.image_position_from_source(source_pos_x, source_pos_y, kwargs_lens)
-        if len(image_positions) < 2:
+        if len(image_positions[0]) < 2:
             return False
 
+        if len(image_positions[0]) == 2:
```

Comment:
do you need to distinguish the case of two images vs arbitrary number of images? You might be able to do it with arrays of arbitrary lengths only

Reply:
since majority of the images number is 2, I believe if it is did in this way not recall numpy may increase the computer speed. @sibirrer